### DISCUSSION TITLE
Question: 请问user-dropdown这个控件有什么显隐机制吗？

### DISCUSSION THREAD

**Discussion by fiorayang** [vbenjs/vue-vben-admin#5234] [General]
### Version

Vben Admin V5

### Describe the bug?

我新建了路由，路由切换的时候，左上角会转圈，然后user-dropdown这个组件就消失了，切回原路由又出来了。路由页面是空页面无任何逻辑，且控制台无任何报错，同时我也查看了相关源码，并没有发现直接控制这个组件显隐的逻辑。

### Reproduction

<img width="766" alt="截屏2024-12-25 17 11 13" src="https://github.com/user-attachments/assets/ac8fb6db-c285-45da-b137-0b87ed448f5f" />

![截屏2024-12-25 17 15 31](https://github.com/user-attachments/assets/16538773-6b9a-476c-a9e4-29789d4871f3)

<img width="660" alt="截屏2024-12-25 17 17 43" src="https://github.com/user-attachments/assets/59f9dfeb-4d95-493d-bea9-13014c2cead5" />

### System Info

```shell
System:
    OS: macOS 14.5
    CPU: (10) arm64 Apple M1 Pro
    Memory: 108.52 MB / 16.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 21.0.0 - /usr/local/bin/node
    npm: 10.2.0 - /usr/local/bin/npm
    pnpm: 9.15.0 - /usr/local/bin/pnpm
  Browsers:
    Chrome: 131.0.6778.205
    Safari: 17.5
```

### Validations

- [X] Read the [docs](https://doc.vben.pro/)
- [X] Ensure the code is up to date. (Some issues have been fixed in the latest version)
- [X] I have searched the [existing issues](https://github.com/vbenjs/vue-vben-admin/issues) and checked that my issue does not duplicate any existing issues.
- [X] Check that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/vbenjs/vue-vben-admin/discussions) or join our [Discord Chat Server](https://discord.gg/8GuAdwDhj6).
- [X] The provided reproduction is a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by anncwb** [comment]
没有这块的逻辑，应该是你自己代码改到了
